lib/galaxy/util/watcher.py

```diff
@@ -154,6 +154,18 @@ def _handle(self, event):
                 callback(path=path)
 
 
+class SimpleFileModifiedEventHandler(FileSystemEventHandler):
```

Comment:
This handler doesn't work for moved files, like when a vim buffer is to the destination path.
Is there any reason the default handler doesn't work for you ?
It works just fine for me.

Reply:
Changed to default handler. There was no big reason for using the other one - seemed like that was reducing complexity at the time. You are right about moved files and md5 hash.